sql/moz-fx-data-shared-prod/subscription_platform_derived/stripe_subscriptions_history_v1/query.sql

```diff
@@ -436,6 +446,7 @@ SELECT
   subscriptions_history_provider_location.provider,
   -- Use the same address hierarchy as Stripe Tax after we enabled Stripe Tax (FXA-5457).
   -- https://stripe.com/docs/tax/customer-locations#address-hierarchy
+  -- Use the first address that is precise enough to include state, if any, otherwise just use country
```

Comment:
I don't think this is advisable, because it doesn't match what Stripe Tax does, which is use the first address in their address hierarchy that's non-empty.

For example, SubPlat could have geo-located a customer in a non-US/Canada country and put that country+postal code in their Stripe shipping address so they're billed based on the subscription plan for that country, and we'd want to go with that even though it doesn't have a state (e.g. we wouldn't want to override that just because they're paying with a card that has a US or Canadian billing address where we could determine the state).

Reply:
Ok I interpreted the wording as the first location that's precise enough to calculate tax but looking at it again, that may be what `requires_location_inputs` is for